web/web.go

```diff
@@ -240,9 +241,16 @@ func MustBuildServerURL(pathPrefix string) string {
 	if err != nil {
 		panic(err)
 	}
-	hostname, err := os.Hostname()
+	hostname, err := Hostname()
 	if err != nil {
 		panic(err)
 	}
 	return fmt.Sprintf("http://%s:%s%s", hostname, port, pathPrefix)
 }
+
+func Hostname() (string, error) {
```

Comment:
*g*
I chose `getHostname` to keep the same pattern like `getTemplate` in the same package.